drivers/amlogic/video_dev/Kconfig

```diff
@@ -14,6 +15,7 @@ config V4L_AMLOGIC_VIDEO2
 	select VIDEO_DEV
 	select VIDEOBUF_GEN
 	select VIDEO_V4L2_COMMON
+	select VIDEOBUF_GEN
```

Comment:
One for `V4L_AMLOGIC_VIDEO` and one for `V4L_AMLOGIC_VIDEO2`. Each can be built separately and both needs VIDEOBUF_GEN.